charts/osm/README.md

```diff
@@ -140,6 +140,7 @@ The following table lists the configurable parameters of the osm chart and their
 | OpenServiceMesh.vault.role | string | `"openservicemesh"` | Vault role to be used by Open Service Mesh |
 | OpenServiceMesh.vault.token | string | `""` | token that should be used to connect to Vault |
 | OpenServiceMesh.webhookConfigNamePrefix | string | `"osm-webhook"` | Prefix used in name of the webhook configuration resources |
+| OpenServiceMesh.webhookTimeout | int | `20` | Webhook timeout seconds for the osm-injector's mutating webhook |
```

Comment:
This should be a part of the injector property.

Reply:
Fixed!